src/algebra/category/Group/basic.lean

```diff
@@ -73,13 +86,23 @@ local attribute [reducible] CommGroup
 @[to_additive]
 instance : has_coe_to_sort CommGroup := infer_instance -- short-circuit type class inference
 
-@[to_additive add_comm_group]
-instance (G : CommGroup) : comm_group G := G.str
+@[to_additive add_comm_group_instance]
+instance comm_group_instance (G : CommGroup) : comm_group G := G.str
 
 @[to_additive] instance : has_one CommGroup := ⟨CommGroup.of punit⟩
 
 @[to_additive] instance : inhabited CommGroup := ⟨1⟩
 
+@[to_additive]
+instance : unique (1 : CommGroup.{u}) :=
+{ default := 1,
+  uniq := λ a, begin cases a, refl, end }
+
+@[to_additive, simp]
+lemma one_apply (G H : CommGroup) (g : G) : (1 : G ⟶ H) g = 1 := rfl
+
+attribute [simp] _root_.AddCommGroup.zero_apply
```

Comment:
Is this a bug in `to_additive`? Or maybe you should put `simp` before it on line 101?

Reply:
Fixed. I'd just noticed that `to_additive` doesn't copy `ext` (noted in #2095), and not tried hard enough here.